src/renderers/native/ReactNativeAttributePayload.js

```diff
@@ -394,9 +394,9 @@ function diffProperties(
       // default: fallthrough case when nested properties are defined
       removedKeys = null;
       removedKeyCount = 0;
-      // $FlowFixMe - We think that attributeConfig is not
-      // CustomAttributeConfiguration at this point so we assume
-      // it must be AttributeConfiguration.
```

Comment:
@gabelevi - Not sure if you or somebody else would want to make this multi-line format work. I would have expected this to work but apparently it doesn't just apply it to the next real code line and presumably just applies it to the next line which in this case was a comment.